violinplot.m

```diff
@@ -65,6 +67,19 @@
             error('Second argument of ''GroupOrder'' optional arg must be a cell of category names')
         end
     end
+    
+    % parse the optional axis handle
```

Comment:
Why is the axis handle checked here, even though `@ishghandle` is already passed to `addParameter`?

Reply:
I just copied the code block from above for checking group order. But `AxisHandle` is also needed in `Violin` arguments, so I left it alone instead of deleting it in `varargin`. I added the check in `addParameter` to keep that consistent with all parameters passed to `Violin` through `varargin`. I figured it would be easier than modifying the `Violin` input arguments to explicitly put `AxisHandle`.

Reply:
I think the check is necessary for `grouporder`, because grouporder is used in violinplot itself. But the `Parent` property is only used in Violin, and therefore does not need to be validated here. Or am I misunderstanding something?

Reply:
The `Parent` property, which is the `AxisHandle` variable, is also used in this function. So I keep the validation here as well, just like the above code block for checking group order.